x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -196,20 +196,15 @@ export const AllCasesList = React.memo<AllCasesListProps>(
       [deselectCases, hasOwner, availableSolutions, owner]
     );
 
-    /**
-     * At the time of changing this from all to delete the only bulk action we have is to delete. When we add more
-     * actions we'll need to revisit this to allow more granular checks around the bulk actions.
-     */
-    const showActions = permissions.delete && !isSelectorView;
-
-    const columns = useCasesColumns({
+    const { columns } = useCasesColumns({
```

Comment:
As discussed, the vertical alignment of these controls is not quite right:

<img width="358" alt="image" src="https://user-images.githubusercontent.com/7405507/193837050-1fb6a507-f2fa-4fb8-9b96-a11951c72e1c.png">

Reply:
Happy for this to be addressed separately as this hasn't been introduced in this PR.

Reply:
Thank you @peteharverson! Let's do it on another PR.